engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetWireframe.java

```diff
@@ -41,14 +44,19 @@ public StateChange getDefaultInstance() {
     @Override
     public RenderPipelineTask generateTask() {
         if (enabled) {
-            return new SetWireframeTask(GL_FILL);
+            return new SetWireframeTask(ENABLED);
         } else {
-            return new SetWireframeTask(GL_LINE);
+            return new SetWireframeTask(DISABLED);
         }
     }
```

Comment:
I recommend that in this method we follow the pattern established in BindFBO and the task should be generated only once.

Given the simplicity of the boolean case I'd suggest to add/modify:

```
private static final RenderPipelineTask enablingTask;
private static final RenderPipelineTask disablingTask;

public RenderPipelineTask generateTask() {
   if (enabled) {
        if(enablingTask == null) {
            enablingTask = new SetWireframeTask(ENABLED);
        }
       return enablingTask;
    } else {
        // similar code as above for disablingTask
    }
}
```